src/ui/components/FogBackground/particle.ts

```diff
@@ -3,8 +3,8 @@ export default class FloatingParticle {
   startY: number;
 
   speed = {
-    x: Math.round(-1 + Math.random() * 2),
-    y: Math.round(-1 + Math.random() * 2)
+    x: (-1 + Math.random() * 2),
```

Comment:
Using `Math.round` improves the performance of the animation. Using floats causes sub-pixel rendering, which requires more resources: https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API/Tutorial/Optimizing_canvas#Avoid_floating-point_coordinates_and_use_integers_instead

Reply:
Ah, good to know - added it back in: 7d48312v